cmd/skupper/skupper_kube_site.go

```diff
@@ -71,6 +71,10 @@ func (s *SkupperKubeSite) Create(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("The minimum value for service-sync-site-ttl is 1 minute")
 	}
 
+	if routerCreateOpts.EnableConsole && !routerCreateOpts.EnableFlowCollector {
```

Comment:
This whole stuff could be moved to NewCmdInit's function under `cmd/skupper/skupper.go`,
right before the `skupperCli.Create` is called, as it is used by both podman and kube sites.

Reply:
moved, thanks